CloudBoltAutomation/Functions/Remove-CBResourceService.ps1

```diff
@@ -0,0 +1,46 @@
+
+function Remove-CBResourceService
+{
+    <#
+    .SYNOPSIS
+    Removes a service resource from CloudBolt.
+
+    .DESCRIPTION
+    The `Remove-CBResourceService` function removes a service resource from CloudBolt. Pass the session to the targeted CloudBolt instance to the `Session` parameter (use `New-CBSession` to create a session). Pass the ID of the service to the `ID` parameter. You can also pipe the ID to this function, or pipe an object with an ID property.
+
+    .EXAMPLE
+    Remove-CBResourceService -Session $session -ID 54
+    
+    Demonstrates how to remove a service resource using its ID.
+    
+    .EXAMPLE
+    Get-CBResourceService -Session $session -ID 65 | Remove-CBResourceService -Session $session    
+
+    Demonstrates how you can pipe service resource objects to `Remove-CBResourceService`.
+
+    .EXAMPLE
+    Remove-CBResourceService -Session $session -ID 49 -WhatIf

```

Comment:
Nit: Is it necessary to document common parameters like `-WhatIf`? I never see it done. They should automatically show up with `Get-Help` as long as `SupportsShouldProcess=$true` is used (I think).